modules/RockRuby.cmake

```diff
@@ -266,6 +266,11 @@ ELSEIF(NOT RUBY_EXTENSIONS_AVAILABLE)
        OUTPUT_VARIABLE RUBY_CFLAGS)
     STRING(REPLACE "\n" "" RUBY_CFLAGS ${RUBY_CFLAGS})
 
+    if ("${CMAKE_CXX_COMPILER_ID}" STREQUAL "Clang")
+        #Clang can't handle this option (04.2014) remove this flag therefore
+        STRING(REPLACE "-fstack-protector-strong" "" RUBY_CFLAGS ${RUBY_CFLAGS})
```

Comment:
> Nevertheless how we handle this, in general the EABIs from clang and gcc
are compatible. So i see no reason why we should not workaround this here.

Me neither